### PR TITLE
Refs #35883 - log useful message if ansible output was censored

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: app/helpers/foreman_ansible/ansible_reports_helper.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def check_mode_log?(log)
     def ansible_module_message(log)
       msg_json = parsed_message_json(log)
       return _("Execution error: #{msg_json['msg']}") if msg_json['failed'].present?
+      return msg_json['censored'] if msg_json['censored'].present?
```
Comment (evgeni) [inline app/helpers/foreman_ansible/ansible_reports_helper.rb:31]: I wonder if we should return the Ansible message, or write something ourself?

**Comment by evgeni** [comment]
somehow GHA didn't trigger :(

File: app/helpers/foreman_ansible/ansible_reports_helper.rb (hunk 1 above)
Comment (nofaralfasi) [reply app/helpers/foreman_ansible/ansible_reports_helper.rb:31]: I think the Ansible message is good. Do you want to ask someone for a third opinion?

Comment (evgeni) [reply app/helpers/foreman_ansible/ansible_reports_helper.rb:31]: nah, if you're good with that, so am I :)

**Comment by nofaralfasi** [comment]
Thanks @evgeni! (third time's a charm)

**Pull request merged**